tests/queries/0_stateless/02534_keyed_siphash.sql

```diff
@@ -345,3 +345,11 @@ CREATE TABLE sipHashKeyed_keys (`a` Map(String, String)) ENGINE = Memory;
 INSERT INTO sipHashKeyed_keys FORMAT VALUES ({'a':'b', 'c':'d'}), ({'e':'f', 'g':'h'});
 SELECT hex(sipHash128ReferenceKeyed((0::UInt64, materialize(0::UInt64)), a)) FROM sipHashKeyed_keys ORDER BY a;
 DROP TABLE sipHashKeyed_keys;
+
+SELECT 'Check bug 3 found fuzzing';
```

Comment:
Instead of enumerating "bug 1", "bug 2", "bug 3" it would be slightly more useful if the comments mention what is actually being tested, e.g. empty arrays/maps.